app/blueprints/fund/templates/fund_details.html

```diff
@@ -0,0 +1,112 @@
+{% extends "base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList -%}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "grants" %}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        {{ govukBackLink({
+          "text": "Back",
+          "href": url_for("fund_bp.view_all_funds")
+        }) }}
+        <h2 class="govuk-heading-m">{{ fund.name_json["en"] }}</h2>
```

Comment:
I think this should be a `govuk-heading-l`

Reply:
Updated